ci-operator/step-registry/configure-registry-storage/set-nutanix-object/configure-registry-storage-set-nutanix-object-commands.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# This script will set image registry to use nutanix object storage. 
+# It need deploy against nutanix platform.
+# Since nutanix object service still not support terraform, so we will use fixed bucket accordint to OCP version. And will clean up data on nutanix web console periodically.
```

Comment:
The data will be saved in aws s3, right?

Reply:
The data will be restored in nutanix object storage https://prismcentral.lts-cluster.nutanix-dev.devcluster.openshift.com:9440/oss/details/3887be40-7d29-4817-486b-4550754433cf/?theme=light#/